Joyce's HW

joyceyskwon opened the pull request:
I have done some works with clearfix to clear floats but I need more
help with the side blurb.

joyceyskwon committed:
created css for corresponding html tags

I have done some works with clearfix to clear floats but I need more
help with the side blurb.

joyceyskwon committed:
added clearfix

hartliddell commented:
@joyceyskwon I made video notes for you. Please watch them. Most students have found video feedback extremely helpful. 

https://drive.google.com/file/d/0B7k_ZQrGrP9caVhiUGhzMGIzLWM/view

joyceyskwon commented:
This was super helpful! Thanks so much Hart!

joyceyskwon committed:
changes in html and css

fixed some typos in html and css!
Still need work with nav icon for @media only screen mode!

joyceyskwon committed:
final changes

I made some changes after watching your video. It was super helpful. I
just forgot to commit..
